python/perf-kernels/model_configs.json

```diff
@@ -0,0 +1,17 @@
+{
+  "llama3_8B": {
+    "head_count": 32,
+    "head_dimension": 128,
```

Comment:
Usually this is backwards - the hidden_size and head_count are provided, and we work out the head_dimension from that.

Here is an example

https://huggingface.co/unsloth/llama-3-8b/blob/main/config.json